code/game/machinery/cryopod.dm

```diff
@@ -192,12 +192,13 @@
 		/obj/item/device/paicard,
 		/obj/item/weapon/gun,
 		/obj/item/weapon/pinpointer,
-		/obj/item/clothing/suit,
```

Comment:
I don't think that would really be all that big of a problem if done right. It would certainly use up a lot more memory if we try to store a lot of DIFFERENT objects, but storing a large number of the same objects just means we have a line in a list consisting of the number, and that object.